packages/cgif/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/dloebl/cgif
+TERMUX_PKG_DESCRIPTION="A fast and lightweight GIF encoding library"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
This line can be dropped, LICENSE is one of the standard filenames that the [install_license script checks for](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L47)